xml/System.Text.Json/JsonTokenType.xml

```diff
@@ -94,7 +94,7 @@
       </ReturnValue>
       <MemberValue>9</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The token type is the JSON literal <see langword="false" />.</summary>
```

Comment:
This should **not** be the .NET language keyword, false. This refers to JSON literal "false" which is subtly different.
The same applies to true/null.

Can we use some annotation to convey that? Maybe `<c></c>`? `langword` is not technically accurate.

@bartonjs, any suggestions?

Reply:
I am not sure if we allow `<c>` in Docs files. It's a good question for @mairaw and @rpetrusha.

We can simply convert this to:

```suggestion
        <summary>The token type is the JSON literal "false".</summary>
```